### PR TITLE
Issue 161 - covid 19 banner

### PR DESCRIPTION
**Pull request by amaxama**
### What
_Brief description of changes._

### Why
_What problem does this pull requests solve?  Reference related issues if appropriate._

### Ensure the following interactions work as expected. Please test using a mobile form factor.
- [x] Tapping on a marker on the map displays information about the marker in a popup.
- [x] Tapping the "Show list of locations" button replaces the map view with a list view.
- [x] Tapping an item in the list replaces the list view with a map view, and navigates the map to the tapped item on the map.
- [x] Tapping one of the ✅'s in the legend filters items on the map and in the list.
- [x] Changing the language to spanish changes things on the page.
- [x] Clicking the Help/Info button opens and closes a menu with information.
- [x] Clicking the X Close button on the Help/Info screen closes the modal.

### Check the app in the following web browsers:
- [x] Chrome
- [ ] Firefox
- [ ] Edge
- [x] Safari

### CODE REVIEW

File: src/i18n/oji.json [hunk 1]
```diff
@@ -74,5 +74,11 @@
   "moment_h": "Daso-diba'igan",
   "moment_d": "Dasogon",
   "moment_M": "Daso-giizis",
-  "moment_y": "Daso-gikinoonowin"
+  "moment_y": "Daso-gikinoonowin",
+  "safety_reminder": "Safety Reminder",
```
Comment (amaxama) [inline src/i18n/oji.json:78]: Placeholders

File: src/i18n/kar.json [hunk 2]
```diff
@@ -77,5 +77,11 @@
   "moment_h": "we.&H.",
   "moment_d": "weHR",
   "moment_M": "wvg",
-  "moment_y": "weH."
+  "moment_y": "weH.",
+  "safety_reminder": "Safety Reminder",
```
Comment (amaxama) [inline src/i18n/kar.json:81]: Placeholders

File: src/i18n/hmn.json [hunk 3]
```diff
@@ -74,5 +74,11 @@
   "moment_h": "1 teev",
   "moment_d": "1 nub",
   "moment_M": "1 hli",
-  "moment_y": "1 xyoo"
+  "moment_y": "1 xyoo",
+  "safety_reminder": "Safety Reminder",
```
Comment (amaxama) [inline src/i18n/hmn.json:78]: placeholders

File: src/i18n/hin.json [hunk 4]
```diff
@@ -62,5 +62,11 @@
   "time_years": "वर्षों",
   "lang_select": "कृपया एक भाषा चुनें",
   "page_title": "ट्विन सिटीज एड मैप",
-  "show_list_button": "स्थानों की सूची दिखाएं"
+  "show_list_button": "स्थानों की सूची दिखाएं",
+  "safety_reminder": "Safety Reminder",
```
Comment (amaxama) [inline src/i18n/hin.json:66]: Placeholders

File: src/i18n/dak.json [hunk 5]
```diff
@@ -74,5 +74,11 @@
   "moment_h": "oáphe",
   "moment_d": "aŋpétu",
   "moment_M": "wí",
-  "moment_y": "ómakha"
+  "moment_y": "ómakha",
+  "safety_reminder": "Safety Reminder",
```
Comment (amaxama) [inline src/i18n/dak.json:78]: placeholders

File: public/index.html [hunk 6]
```diff
@@ -5,6 +5,7 @@
   <title>Twin Cities Mutual Aid</title>
   <meta name="viewport" content="initial-scale=1,maximum-scale=1,user-scalable=no" />
   <link rel="icon" href="favicon.ico">
+  <link href="https://fonts.googleapis.com/css?family=Material+Icons|Material+Icons+Outlined|Material+Icons+Two+Tone|Material+Icons+Round|Material+Icons+Sharp" rel="stylesheet">
```
Comment (amaxama) [inline public/index.html:8]: Pulling in Material Icons cause it's much easier than having to create new pngs or svgs anytime we want an icon. Also styling icons is way easier this way

File: src/index.js [hunk 7]
```diff
@@ -146,6 +149,10 @@ function truthy(str) {
   return _.includes(['TRUE', 'YES', 'T', 'Y'], normalizedStr);
 }
 
+function closeCovidBanner() {
+  document.getElementById('covid-banner').style.display = "none"
```
Comment (mc-funk) [inline src/index.js:153]: I'm not an a11y expert by any means, but would it be worth adding `.setAttribute("aria-hidden", true);` for this?

File: src/styles/components/covid-banner.css [hunk 8]
```diff
@@ -0,0 +1,183 @@
...
+.covid-banner-reminder {
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+}
+
+.covid-warning-icon {
+    /* hide alt text if image is missing */
```
Comment (mc-funk) [inline src/styles/components/covid-banner.css:17]: Are these TODOs or are the margins accomplishing this?

**Review by mc-funk — APPROVED** [review]
This looks good! Personally, I wouldn't be opposed to merging before we have all languages and adding them incrementally, since the visuals are there too, but really up to your judgment.

File: src/styles/components/covid-banner.css (hunk 8 above)
Comment (amaxama) [reply src/styles/components/covid-banner.css:17]: Hmmm, yeah I think that may have been accidentally left around. Good catch, thanks!

File: src/index.js (hunk 7 above)
Comment (amaxama) [reply src/index.js:153]: I also don't know too too much but did a little looking and that seems reasonable, yes! Will add it!

**Comment by amaxama** [comment]
@mc-funk Updated based on your comments. And I'm fine with that if you are for translations. I made a follow-up issue to come back to so it doesn't get forgotten.

**Review by mc-funk — APPROVED** [review]
Awesome, thanks!

**Pull request merged**